test/Groups/operations.jl

```diff
@@ -86,7 +86,7 @@ end
    x=matrix(F,4,4,[1,z,0,0,0,1,z^2,z,z,0,0,1,0,0,z+1,0])
    y=x+transpose(x)
    @test is_symmetric(y)
-   @test is_hermitian_matrix(x+conjugate_transpose(x))
+   @test is_hermitian(x+conjugate_transpose(x))
    @test is_skewsymmetric_matrix(y)
```

Comment:
I am fine with this change, but then perhaps we should also do the same for `is_skewsymmetric` ?

Reply:
Please see https://github.com/oscar-system/Oscar.jl/issues/2555 for this.
There are two semantically different implementations that would be merged by renaming `is_skewsymmetric_matrix`.